codec-http2/src/test/java/io/netty/handler/codec/http2/Http2TestUtil.java

```diff
@@ -119,7 +119,7 @@ public static HpackEncoder newTestEncoder() {
 
     public static HpackEncoder newTestEncoder(boolean ignoreMaxHeaderListSize,
                                               long maxHeaderListSize, long maxHeaderTableSize) throws Http2Exception {
-        HpackEncoder hpackEncoder = new HpackEncoder();
```

Comment:
This change does not look right to me

Reply:
`ignoreMaxHeaderListSize` was previously ignored, so I left it as is.  The other values are the defaults, except for the `true`, which is to preserve the hard coded hpack values.

Reply:
Ah ok 👌